pkg/termio/progress.go

```diff
@@ -129,7 +129,14 @@ func (p *ProgressBar) doPrint() {
 	}
 
 	barWidth := uint(termWidth)
-	digits := int(math.Log10(float64(max))) + 1
+	var digits int
+	// Log10(0) is undefined
+	if max > 0 {
```

Comment:
I think we can make this a little bit shorter.

Could you leave the initial assignment as is and only add three lines to check max < 1 and then set the value to 1.